### DISCUSSION TITLE
How to add markdown to a field?

### DISCUSSION THREAD

**Discussion by MrHinsh** [estruyf/vscode-front-matter#865] [Q&A] [answered]
I use markdown in all of my `content` variables in the front matter. I really don't see why one would want HTML in there when its in a *.md file and the rest is markdown. 

How can I have a multi-line textbox but prevent any HTML from being added?

**Comment by estruyf** [comment, marked as answer]
Hi @MrHinsh, at the moment, only HTML is supported, as it was based on the original request. It would be an enhancement to the string field also to support Markdown.

**Reply by MrHinsh** [reply]
I understand... logically it should produce markdown. no?

**Reply by estruyf** [reply]
I've created the following issue: https://github.com/estruyf/vscode-front-matter/issues/866
